diff -up kipi-plugins-0.8.0-patched/cmake/modules/FindGdk.cmake.gdk-path-fix kipi-plugins-0.8.0-patched/cmake/modules/FindGdk.cmake
--- kipi-plugins-0.8.0-patched/cmake/modules/FindGdk.cmake.gdk-path-fix	2009-11-01 23:51:19.000000000 +0100
+++ kipi-plugins-0.8.0-patched/cmake/modules/FindGdk.cmake	2015-02-26 14:04:39.760322559 +0100
@@ -23,7 +23,7 @@ else (GDK_INCLUDE_DIR AND GDK_LIBRARIES)
     set(GDK_DEFINITIONS ${_GDKCflags})
   endif(NOT WIN32)
 
-  FIND_PATH(GDK_INCLUDE_DIR gdk-pixbuf/gdk-pixbuf.h /usr/include/gtk-2.0
+  FIND_PATH(GDK_INCLUDE_DIR gdk-pixbuf/gdk-pixbuf.h /usr/include/gdk-pixbuf-2.0
     ${_GDKIncDir}
   )
   
diff -up kipi-plugins-0.8.0-patched/ipodexport/IpodExportDialog.cpp.gdk-path-fix kipi-plugins-0.8.0-patched/ipodexport/IpodExportDialog.cpp
--- kipi-plugins-0.8.0-patched/ipodexport/IpodExportDialog.cpp.gdk-path-fix	2015-02-26 14:10:29.230322304 +0100
+++ kipi-plugins-0.8.0-patched/ipodexport/IpodExportDialog.cpp	2015-02-26 14:07:59.707322335 +0100
@@ -63,6 +63,7 @@
 
 // Libgdk includes.
 
+#undef signals
 extern "C"
 {
 #include <gdk-pixbuf/gdk-pixbuf.h>
